acquisitions/src/main/resources/thunderjet/cross-modules/features/rollover-with-no-settings.feature

```diff
@@ -0,0 +1,162 @@
+    * def createOrder = read('classpath:thunderjet/mod-orders/reusable/create-order.feature')
+    * def createOrderLine = read('classpath:thunderjet/mod-orders/reusable/create-order-line.feature')
+    * def openOrder = read('classpath:thunderjet/mod-orders/reusable/open-order.feature')
+    * def validateOrderLineEncumbranceLinks = read('classpath:thunderjet/mod-orders/reusable/validate-order-line-encumbrance-links.feature')
+
+  @Positive
+  Scenario: Rollover with no settings
+    * def fromYear = call getCurrentYear
+    * def toYear = parseInt(fromYear) + 1
+    * def fiscalYearId1 = call uuid
+    * def fiscalYearId2 = call uuid
+    * def ledgerId = call uuid
+    * def fundId = call uuid
+    * def budgetId1 = call uuid
+    * def budgetId2 = call uuid
+
+    * def orderId1 = call uuid
+    * def orderId2 = call uuid
+    * def orderId3 = call uuid
+    * def orderId4 = call uuid
+
+    * def poLineId1 = call uuid
+    * def poLineId2 = call uuid
+    * def poLineId3 = call uuid
+    * def poLineId4 = call uuid
+
+    * def ongoingObj = { "interval": 123, "isSubscription": true, "renewalDate": "2022-05-08T00:00:00.000+00:00" }
+
+    * def rolloverId = call uuid
+
+    * table orders
+      | id       | orderId  | orderType  | reEncumber | ongoing    |
+      | orderId1 | orderId1 | 'One-Time' | true       | null       |
+      | orderId2 | orderId2 | 'One-Time' | false      | null       |
+      | orderId3 | orderId3 | 'Ongoing'  | true       | ongoingObj |
+      | orderId4 | orderId4 | 'Ongoing'  | false      | ongoingObj |
+
+    * table orderLines
+      | id        | orderId  | listUnitPrice |
+      | poLineId1 | orderId1 | 10.0          |
+      | poLineId2 | orderId2 | 10.0          |
+      | poLineId3 | orderId3 | 10.0          |
+      | poLineId4 | orderId4 | 10.0          |
+
+    ### 1. Create fiscal years and associated ledgers
+    * def periodStart1 = fromYear + '-01-01T00:00:00Z'
+    * def periodEnd1 = fromYear + '-12-30T23:59:59Z'
+    * def v = call createFiscalYear { id: '#(fiscalYearId1)', code: 'TESTFYA0012', periodStart: '#(periodStart1)', periodEnd: '#(periodEnd1)', series: 'TESTFYA' }
+
+    * def periodStart2 = toYear + '-01-01T00:00:00Z'
+    * def periodEnd2 = toYear + '-12-30T23:59:59Z'
+    * def v = call createFiscalYear { id: '#(fiscalYearId2)', code: 'TESTFYA0013', periodStart: '#(periodStart2)', periodEnd: '#(periodEnd2)', series: 'TESTFYA' }
+    * def v = call createLedger { id: '#(ledgerId)', fiscalYearId: '#(fiscalYearId1)' }
+
+    ### 2. Create fund and budgets
+    * configure headers = headersAdmin
+    * def v = call createFund { id: '#(fundId)', code: '#(fundId)', ledgerId: '#(ledgerId)' }
+    * def v = call createBudget { id: '#(budgetId1)', fundId: '#(fundId)', fiscalYearId: '#(fiscalYearId1)', allocated: 1000, status: 'Active' }
+    * def v = call createBudget { id: '#(budgetId2)', fundId: '#(fundId)', fiscalYearId: '#(fiscalYearId2)', allocated: 1000, status: 'Active' }
+
```

Comment:
we should revert back to headers users after creation of fund and budget
* configure headers = headersAdmin